x/supplier/keeper/msg_server_unstake_supplier_test.go

```diff
@@ -240,10 +240,9 @@ func TestMsgServer_UnstakeSupplier_CancelUnbondingIfRestaked(t *testing.T) {
 
 	expectedSupplier.UnstakeSessionEndHeight = sharedtypes.SupplierNotUnstaking
 	expectedSupplier.Stake = stakeMsg.GetStake()
-	// Set the deactivation height of the current service configuration to the next block after session end
-	// This mimics the behavior of the staking process, which effectively marks all
-	// the previous service configurations as deactivated.
-	expectedSupplier.ServiceConfigHistory[0].DeactivationHeight = sessionEndHeight + 1
+	// When restaking without providing services, existing service configs are preserved unchanged
```

Comment:
Can you make sure this is documented in two place:
- The CLI --help
- The docs